### DISCUSSION TITLE
Cannot install drivers

### DISCUSSION THREAD

**Discussion by frankenkj** [arkypita/LaserGRBL#992] [Q&A]
Win 10 x64 machine -

Laser Grbl installed and launches fine - when installing driver CH340 driver, it extracts and finds the inf, but errors out when installing. 
any ideas why ?
![driver error](https://user-images.githubusercontent.com/64806615/81127937-67f5ec80-8f05-11ea-94d8-112580c0bd47.jpg)

**Comment by StuartB4** [comment]
You can try this version if you want. Just extract it and run it outside of LaserGRBL.
[CH341SER.ZIP](https://github.com/arkypita/LaserGRBL/files/4584383/CH341SER.ZIP)

**Comment by frankenkj** [comment]
getting the same error

![fail](https://user-images.githubusercontent.com/64806615/81128815-df2c8000-8f07-11ea-8cae-7d0d11c06f92.jpg)

**Comment by StuartB4** [comment]
Have you been in to windows Device manager to see if there is one already there, and a comport number.
![565666](https://user-images.githubusercontent.com/16980026/81129602-c25e6f80-8f3c-11ea-8f58-94010256da10.jpg)

**Comment by arkypita** [comment]
Maybe you need administrator privilege?
Try right-click, run as administrator

**Comment by frankenkj** [comment]
> 
> 
> Maybe you need administrator privilege?
> Try right-click, run as administrator

I tried that and same result.


Interestingly tho, without the dirvers installed I can control the laser via Laser GRBL and Lightburn.  Maybe it's  a non-issue ?

**Comment by arkypita** [comment]
> Interestingly tho, without the dirvers installed I can control the laser via Laser GRBL and Lightburn. Maybe it's a non-issue ?

Maybe you have already installed them. Or you don't need them.

CH340 driver is needed only if your board use a ch340 chip for usb serial communication.
Different board types can use different chip.
